notifier/typetalk/client.go

```diff
@@ -0,0 +1,72 @@
+package typetalk
+
+import (
+	"errors"
+	"os"
+	"strconv"
+
+	"github.com/mercari/tfnotify/terraform"
+	typetalk "github.com/nulab/go-typetalk/typetalk/v1"
+)
+
+// EnvToken is Typetalk API Token
+const EnvToken = "TYPETALK_TOKEN"
+
+// Client represents Typetalk API client.
+type Client struct {
+	*typetalk.Client
+	Config Config
+	common service
+	Notify *NotifyService
+	API    API
+}
+
+// Config is a configuration for Typetalk Client
+type Config struct {
+	Token    string
+	Title    string
+	TopicID  string
+	Message  string
+	CI       string
+	Parser   terraform.Parser
+	Template terraform.Template
+}
+
+type service struct {
+	client *Client
+}
+
+// NewClient returns Client initialized with Config
+func NewClient(cfg Config) (*Client, error) {
+	token := cfg.Token
+	if token == EnvToken {
```

Comment:
In GitHub client,

deal with `$` dollar explicitly. How about following this in Typetalk client also? 

https://github.com/mercari/tfnotify/blob/c757bd916a36e2d124180900aa677741d8490bc8/notifier/github/client.go#L62-L65

Reply:
https://github.com/mercari/tfnotify/pull/27/files#r247919562